src/Components/CreateAntFields/CreateAntFields.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { DatePicker, Form, Input, Select, InputNumber } from "antd";
+import { FieldContainer } from './styled';
+
+const FormItem = Form.Item;
+const { Option } = Select;
+const { TextArea } = Input;
+
+const CreateAntField = AntComponent => ({
```

Comment:
This it is not component, but function that return component.
Name should start from lowercase.

Reply:
fixed